src/einsteinpy/symbolic/ricci.py

```diff
@@ -96,3 +154,74 @@ def symbols(self):
 
         """
         return self.syms
+
+
+class RicciScalar:
+    """
+    Class for defining Ricci Scalar
+    """
+
+    def __init__(self, metric_tensor, ricci_tensor):
```

Comment:
`def __init__(self, expression, syms, parent_metric=None)`
for init, it would be like any other GR tensor, just missing the `config` argument because it is scalar(no indices)

Reply:
`expression` means a simple sympy expression, not array, not list.